ADMVEDU95/ADMVEDU95/Routers/BaseRouterProtocol.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  BaseRouterProtocol.swift
+//  ADMVEDU95
+//
+//  Created by Satishur, Oleg on 30.04.2021.
+//
+
+import UIKit
+
+protocol BaseRouterProtocol {
+    func navigateToHome()
+    func navigateToAuth()
+}
+
+extension BaseRouterProtocol {
+    var sceneDelegate: SceneDelegate? {
```

Comment:
why do we need it? no navigation should be placed in scene delegate except starting router